src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -226,19 +303,25 @@ private void updateClosedLoopSlot() {
     };
   }
 
+  /**
+   * If the elevator is calibrated, set m_isCalibrated to true
+   */
```

Comment:
maybe clarify what this means, that the elevator can use the modes or whatever, again not about how it happens (setting a variable) but about what it does